File: Parte_1/codigos/micro07.py
def micro07():
    
	numero  = 0
    programa = 1
	opc = ""

	while programa == 1:
		print("Digite um numero: ",end = "")
		numero = int(raw_input())

		if numero > 0:
			print("Positivo")
		else:
			if numero == 0:
				print("O numero e igual a 0")
			if numero < 0:
				print("Negativo")

		print("Deseja Finalizar? (S/N) ",end = "")
		opc = raw_input()
		if opc == "S":
			programa = 0

micro07()

File: Parte_1/codigos/micro07.c
#include <stdio.h>

int main(){
	int programa, numero;
	char opc;

	programa = 1;
	while( programa == 1){
		printf("Digite um número: ");
		scanf("%d",&numero);

		if(numero > 0){
			printf("Positivo");
		}
		else{
			if(numero == 0){
				printf("O número é igual a  0");
			}
			if (numero < 0){
				printf("Negativo");
			}
		}
		getchar();
		printf("Deseja finalizar? (S/N) ");
		scanf("%c", &opc);
		if(opc == 'S'){
			programa = 0;
		}
	}
}
